specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/trafficmanager.json

```diff
@@ -1137,6 +1226,19 @@
       },
       "description": "Class which is a sparse representation of a Traffic Manager endpoint."
     },
+    "HeatMapLocation": {
```

Comment:
Now you split everything, this type should be used somewhere.

Reply:
ar... forgot to update this def name, should be "TrafficLocation" - i'm assuming when I use "$ref": "#/definitions/TrafficLocation", this "TrafficLocation" will be applied.